### PR TITLE
🧹 Remove duplicate bundle identifier lint warning

### PR DESCRIPTION
**Pull request by danyf90**
In microfeatures architecture, it is common to assign the same bundle identifier to the main app and to the features demo, in order to be able to run them on real devices, hence the linting has been removed

Discussed [here](https://tuistapp.slack.com/archives/GGB99EP1A/p1640711532008400)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -98,3 +98,5 @@ node_modules
 
 Tuist/Dependencies/SwiftPackageManager
 Tuist/Dependencies/graph.json
+
+.vscode
```
Comment (pepicrft) [inline .gitignore:102]: What's the reason to add this one? The `.vscode` directory contains settings that belong to the repository and therefore I think we think we should revert this change.

**Review by pepicrft — APPROVED** [review]
Once the `.vscode` comment is addressed and CI is green we can merge.

File: .gitignore (hunk 1 above)
Comment (danyf90) [inline .gitignore:102]:
```suggestion
```

Comment (danyf90) [reply .gitignore]: Ops, I have seen a new created file `.vscode/launch.json` (probably because of the swift plugin for VSCode) and I though it was not supposed to be committed

Comment (danyf90) [reply .gitignore]: Maybe we should add the `launch.json` as well (possibily in a separate PR)

**Review by luispadron — APPROVED** [review]
Yay 🧹
